dev-python/pyilmbase-2.2.0 and media-gfx/alembic-1.7.4

waebbl opened the pull request:
Request to add two new packages.
The dev-python/pyilmbase package implements python bindings for the media-libs/ilmbase-2.2.0 library. It's distributed at http://www.openexr.com together with ilmbase and openexr libs.

The media-gfx/alembic package is a framework for storing and sharing scene data. This can be used in blender from version 2.78 upwards, although it's currently not support in the official blender builds.
It builds for me, and blender find's and compiles it and runs fine, if you ask it to include alembic support.
I added the license file as Alembic, because I'm unsure whether the license statements are already supported by one of the licenses.
Feedback welcome.

waebbl commented:
Cleanup of history, removing commits not belonging to this package.

jonasstein commented on licenses/Alembic:
```diff
@@ -0,0 +1,116 @@
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THE OPEN 
+SOURCE COMPONENTS, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
+
+Boost C++ Libraries 
```
We use **one file per license**. Please suggest the new license to the license team.

waebbl replied:
I was thinking I removed the third party licenses. Will do this, asap.

@gentoo/licenses Can someone please help me to review this license for validity with gentoo?

waebbl commented:
Removed third party licenses from license file and squashed initial commits of the packages with the commits for proxy maint application.
@gentoo/licenses Can someone please help me verify this license?

jonasstein commented on licenses/Alembic:
```diff
@@ -0,0 +1,63 @@
+TM & © 2009-2015 Lucasfilm Entertainment Company Ltd. or Lucasfilm Ltd.
+All rights reserved.
```
I am not sure what line 1-2 do here, but at line 4 BSD-3 clause starts: https://opensource.org/licenses/BSD-3-Clause
How to proceed with the Attachment?

jonasstein commented:
The License problems are solved. Someone can test and merge it.
